examples/network/imageSelected/imageSelected.html

```diff
@@ -0,0 +1,70 @@
+<html>
+<head>
+  <title>Network | Selected/Unselected Image</title>
+
+  <script type="text/javascript" src="../../../dist/vis.js"></script>
+  <link href="../../../dist/vis-network.min.css" rel="stylesheet" type="text/css" />
+
+  <style type="text/css">
+    body {
+      font: 10pt arial;
+    }
+    #mynetwork {
+      width: 600px;
+      height: 600px;
+      border: 1px solid lightgray;
+      background-color:#333333;
+    }
+  </style>
+  <script src="../../googleAnalytics.js"></script>
```

Comment:
Please remove the analytics. We are not using them since f38452130a92273babb0efbbf4a0c2c5059f93dd